frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+
+import { FLAGS, connectToFlags, flagPending } from '../../features';
+import { Firehose, PageHeading, StatusBox } from '../utils';
+import { CatalogTileViewPage } from '../catalog-items';
+import { referenceForModel } from '../../module/k8s';
+import { getServiceClassIcon, getServiceClassImage } from '../catalog-item-icon';
+import { PackageManifestModel } from '../../models';
+
+class MarketplaceListPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [],
+    };
+  }
+  componentDidUpdate(prevProps) {
+    const {packagemanifests, namespace} = this.props;
+    if (packagemanifests !== prevProps.packagemanifests ||
+      namespace !== prevProps.namespace) {
+      this.createMarketplaceData();
+    }
+  }
+  createMarketplaceData() {
+    const {packagemanifests, loaded} = this.props;
+    let packageManifestItems = null;
+    if (!loaded) {
+      return;
+    }
+    if (packagemanifests) {
+      packageManifestItems = this.normalizePackageManifests(packagemanifests.data, 'PackageManifest');
+    }
+    const items = _.sortBy([...packageManifestItems], 'tileName');
+    this.setState({items});
+  }
+  normalizePackageManifests(packageManifests, kind) {
+    const activePackageManifests = _.filter(packageManifests, packageManifest => {
+      return !packageManifest.status.removedFromBrokerCatalog;
+    });
+    return _.map(activePackageManifests, packageManifest => {
+      const tileName = packageManifest.metadata.name;
+      const iconClass = getServiceClassIcon(packageManifest);
+      const tileImgUrl = getServiceClassImage(packageManifest, iconClass);
+      const tileIconClass = tileImgUrl ? null : iconClass;
+      const tileDescription = packageManifest.metadata.description;
+      const tileProvider = packageManifest.metadata.labels.provider;
+      const tags = packageManifest.metadata.tags;
+      const { name, namespace } = packageManifest.metadata;
+      const href = null;
+      return {
+        obj: packageManifest,
+        kind,
+        tileName,
+        tileIconClass,
+        tileImgUrl,
+        tileDescription,
+        tileProvider,
+        href,
+        tags,
+      };
+    });
+  }
+  render() {
+    const { loaded, loadError } = this.props;
+    const { items } = this.state;
+    return <StatusBox data={items} loaded={loaded} loadError={loadError} label="Resources">
+      <CatalogTileViewPage items={items} />
```

Comment:
Likely reusing that page will not work. I assume the categorization and filtering options will be determined differently. For now, I'd simply render th list of tiles here rather than include this page component.